src/components/atoms/SearchBar/index.jsx

```diff
@@ -1,6 +1,17 @@
 import SearchBar from './SearchBar';
 import SearchBarLong from './SearchBarLong';
 
+import PropTypes from 'prop-types';
+
+SearchBar.propTypes = {
+  type: PropTypes.oneOf(['long']),
```

Comment:
Here also prop `type` and `long` are optional prop,

Reply:
how do i do that